p2p/net/swarm/swarm_dial.go

```diff
@@ -65,6 +67,17 @@ var (
 	ErrGaterDisallowedConnection = errors.New("gater disallows connection to peer")
 )
 
+// ErrQUICDraft29 wraps [ErrNoTransport] and provide a more meaningful error message while
```

Comment:
```suggestion
// ErrQUICDraft29 wraps ErrNoTransport and provide a more meaningful error message
```

Reply:
BTW the `[]` around `ErrNoTransport` makes the string `ErrNoTransport` on pkg.go.dev a clickable link to `ErrNoTransport`'s definition

Reply:
Docs here https://golang.org/doc/comment

Reply:
I use gopls which don't do that.
I think it would take an hour or two max to write a linter which finds all potentially applications of this.
Someone might already have done so.